src/app/core/document-items.schema.json

```diff
@@ -0,0 +1,54 @@
+{
+    "$schema": "http://json-schema.org/draft-07/schema#",
```

Comment:
My only hesitation moving from TS to JSON for the document items is I wanted to make sure there was still dev-time validation.  This JSON schema accomplishes that.